src/applications/vaos/utils/constants.js

```diff
@@ -18,13 +18,27 @@ export const TIME_TEXT = {
   'No Time Selected': '',
 };
 
-export const PURPOSE_TEXT = {
+export const REASON_RADIO_BUTTONS = {
```

Comment:
Peggy suggested that the radio button labels be more conversational.  Separated the radio button labels and the text that will show up on the review/confirmation pages into two different constants.